bases/renku_data_services/data_api/config.py

```diff
@@ -172,6 +175,9 @@ def from_env(cls):
             user_store = KcUserStore(keycloak_url=keycloak_url, realm=keycloak_realm)
             gitlab_client = GitlabAPI(gitlab_url=gitlab_url)
 
+        max_pinned_projects = int(os.environ.get("MAX_PINNED_PROJECTS", "0"))
```

Comment:
Just add a default value here so that if nothing is specified we can still function. I picked 5 as the default but maybe 10 is better? Also if you want you can add the default straight into the UserPreferencesConfig class and then not even check the env variables. If we ever want to control this more we can add the env variable part later.

Reply:
Also there are some conflicts in this PR.

Reply:
The env variable part is already configured from `values.yaml` -> https://github.com/SwissDataScienceCenter/renku/pull/3338/files#diff-6e7239bb43b5eb3cf24e9c4106116bee13a43c74580add111af16bb6d82cd9efR1514